src/js/components/AppPageComponent.jsx

```diff
@@ -157,7 +157,16 @@ var AppPageComponent = React.createClass({
     });
   },
 
-  onScaleAppError: function (errorMessage) {
+  onScaleAppError: function (errorMessage, statusCode, instances) {
+    if (statusCode === 409) {
+      var appId = this.state.appId;
+      const dialogId = DialogActions.
+        confirm("App scaling failed.Do you want to try forcing?");
+      DialogStore.handleUserResponse(dialogId, function () {
+        AppsActions.scaleApp(appId, instances, true);
+      });
+      return;
```

Comment:
I think we need to only return (and with an explicit value) dependent on what the user answers to the dialog here.
What happens if the user clicks `cancel`?